src/test/java/org/jenkinsci/plugins/ghprb/GhprbPullRequestMergeTest.java

```diff
@@ -282,7 +282,7 @@ public void testTriggerMerge() throws Exception {
         assertThat(merger.perform(build, null, listener)).isEqualTo(true);
 
         setupConditions(adminLogin, committerName, committerEmail, nonTriggerPhrase);
-        assertThat(merger.perform(build, null, listener)).isEqualTo(false);
+        assertThat(merger.perform(build, null, listener)).isEqualTo(true);
```

Comment:
The intentions here are to verify that the merge didn't happen.  How would you still verify that the merge wasn't attempted?